app/src/androidTest/java/com/github/factotum_sdp/factotum/utils/PreferencesSetting.kt

```diff
@@ -6,13 +6,26 @@ import androidx.test.espresso.Espresso.openActionBarOverflowOrOptionsMenu
 import androidx.test.espresso.action.ViewActions.click
 import androidx.test.espresso.assertion.ViewAssertions.matches
 import androidx.test.espresso.matcher.ViewMatchers.isDisplayed
+import androidx.test.espresso.matcher.ViewMatchers.withId
 import androidx.test.espresso.matcher.ViewMatchers.withText
 import com.github.factotum_sdp.factotum.R
+import kotlinx.coroutines.delay
+import kotlinx.coroutines.runBlocking
+import org.hamcrest.core.AllOf.allOf
 
 object PreferencesSetting {
+
+    private const val WAIT_TIME_BUTTON = 500L
+
     fun enableTouchClick() {
         openActionBarOverflowOrOptionsMenu(ApplicationProvider.getApplicationContext())
-        onView(withText(R.string.rb_label_touch_click)).check(matches(isDisplayed()))
+
+        // Add an explicit wait
+        runBlocking{delay(WAIT_TIME_BUTTON)}
```

Comment:
Not sure that one is needed, it never fails in DRecordDetailsFragmentTest on the CI. I don't think it fixes some flakyness. Could you get it out ?

Reply:
One of my test had issue see the "Touch click" button sometime, and it seems to solve it